### PR TITLE
Add `uid` to `compute_Application` + use common function

### PR DESCRIPTION
**Pull request by tovbinm**
## Description

Add 'uid' to 'compute_Application' + use common function

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

### CODE REVIEW

**Review by thomas-gerber — CHANGES_REQUESTED** [review]
@tovbinm I think your are missing https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/bitbucket/deployments.ts#L51 / https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/bitbucket/common.ts#L16 (and possibly in other sources as well.

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts [hunk 1]
```diff
@@ -36,10 +37,18 @@ export class Pipelines extends AzurePipelineConverter {
       });
     }
 
+    const applicationMapping = this.applicationMapping(ctx);
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts:40]: @thomas-gerber @cjwooo @willmarks should we be writing `compute_Application` entries from Azure pipelines converter? same question for Bitbucket deployments converter?

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts [hunk 2]
```diff
@@ -36,10 +37,18 @@ export class Pipelines extends AzurePipelineConverter {
...
+
     for (const runItem of pipeline.runs) {
-      const applicationMapping = this.applicationMapping(ctx);
-      const application =
-        (applicationMapping && applicationMapping[runItem.name]) ?? null;
+      let application = null;
+      if (runItem?.name) {
+        const mappedApp = applicationMapping[runItem.name];
+        application = Common.computeApplication(
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts:46]: @thomas-gerber @cjwooo @willmarks should we be writing `compute_Application` entries from Azure `pipelines` converter? same question for Bitbucket `deployments` converter?

Comment (tovbinm) [reply destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts:46]: Right now we are only writing out the refs from the deployments (but not the `compute_Application` entries)

Comment (cjwooo) [reply destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts:46]: Yea we should.

**Review by thomas-gerber — APPROVED** [review]
Great! 
Also appreciated that applications are written for Bitbucket / Azure even if mappings are not provided.

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts (hunk 2 above)
Comment (tovbinm) [reply destinations/airbyte-faros-destination/src/converters/azurepipeline/pipelines.ts:46]: ok, I adding it.

**Pull request merged**